src/StatsFuns.jl

```diff
@@ -228,6 +228,18 @@ export
     srdistinvccdf,       # inverse-ccdf of studentized range distribution
     srdistinvlogcdf,     # inverse-logcdf of studentized range distribution
     srdistinvlogccdf,    # inverse-logccdf of studentized range distribution
+    
+    # distrs/wilcox
```

Comment:
Can you add these to the README as well? (I notice now that we forgot to do that in #173, maybe fix it here as well or open a separate PR for `signrank`?)